File: src/luogu/p3954/Main.cpp
// https://www.luogu.com.cn/problem/P3954
#include <cstdio>

int main() {
  int a, b, c;
  scanf("%i%i%i", &a, &b, &c);
  printf("%i\n", (int)(a * 0.2 + b * 0.3 + c * 0.5));
  return 0;
}

File: src/luogu/p3954/Main.kt
// https://www.luogu.com.cn/problem/P3954
fun main() {
  val i = readLine()!!.trim() { it <= ' ' }.split(" ").map { it.toInt() }
  println((i[0] * 0.2 + i[1] * 0.3 + i[2] * 0.5).toInt())
}
